Dentist.h

```diff
@@ -0,0 +1,45 @@
+#pragma once 
+
+#include "LeakedObjectDetector.h"
+#include <iostream>
+
+struct Dentist 
+{
+    Dentist();
+    ~Dentist();
+   
+    std::string dentistName = "Mike";
+    int patientsPerWeek = 40;
+    int costDisposableEquipment = 200;
+    int chargePerPatient = 800;
+    int numStaff;
+
+    struct Person
+    {
+        Person();
+        ~Person();
+       
+        std::string name;
+        int age;
+        std::string gender;
+        
+        std::string insuranceCompany = "No Insurance";
+        int dateOfLastCheck = 112022;
+
+        std::string getInsuranceCompany(std::string insurance); 
+        int dateOfNextDentalAppointment(int lastCheck); 
+        std::string getGender() const; 
+        void numberOfAppointments(int howOld);
+        void printPersonMemberInfo() const;
+    };
+
+    int returnLastCheck(std::string name);
+    float feeForService(const std::string& treatmentClassification);
+    int payStaff(); 
+    int costPerPatient(int num);
+    void printDentistMemberInfo();
```

Comment:
Some of these function names do not contain verbs.
Name things what they are, name functions based on the action they perform.
- [ ] revise